### PR TITLE
doc: SUIT DFU customize guide fixes for 2.8.0

### PR DESCRIPTION
**Pull request by ahasztag**
This commit updates this part of the documentation taking into account changes for 2.8.0 release

### CODE REVIEW

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst [hunk 1]
```diff
@@ -61,7 +61,7 @@ Build system configuration
 **************************
 
 By default the build system generates SUIT envelopes using predefined manifest templates provided by Nordic Semiconductor.
-These templates can be found in :file:`modules/lib/suit-generator/ncs`, and are suitable for standard development needs.
+These templates can be found in :file:`nrf/config/suit/templates` and are suitable for standard development needs.
```
Comment (maciejpietras) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst:64]:
```suggestion
These templates can be found in :file:`nrf/config/suit/templates` and are suitable for typical development needs.
```

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst [hunk 2]
```diff
@@ -264,6 +265,7 @@ With the Python dictionary you are able to, for example:
 * Obtain the size of partition with ``application['dt'].chosen_nodes['zephyr,code-partition'].regs[0].size``
 * Get the pair of URI name and the binary path by using ``'#{{ application['name'] }}': {{ application['binary'] }}``
 * Get the root manifest sequence number with ``suit-manifest-sequence-number: {{ APP_ROOT_SEQ_NUM }}``
+* Gey thr vendor name by using ``application['config']['CONFIG_SUIT_MPI_APP_LOCAL_1_VENDOR_NAME']``
```
Comment (maciejpietras) [inline doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst:268]:
```suggestion
* Get the vendor name by using ``application['config']['CONFIG_SUIT_MPI_APP_LOCAL_1_VENDOR_NAME']``
```

**Review by maciejpietras — CHANGES_REQUESTED** [review]
Small typos

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst [hunk 3]
```diff
@@ -7,7 +7,7 @@ How to customize the SUIT DFU process
    :local:
    :depth: 2
 
-Nordic Semiconductor provides a Software Update Internet of Things (SUIT) sample (:ref:`ug_nrf54h20_suit_intro`) which uses predefined configurations in the build system.
+Nordic Semiconductor provides a :ref:`Software Update Internet of Things (SUIT) sample <nrf54h_suit_sample>` which uses predefined configurations in the build system.
```
[the remark this answers is not in the document]
Comment (ahasztag) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst]: Fixed

**Review by tomchy — APPROVED** [review]

**Review by adsz-nordic — APPROVED** [review]

File: doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst (hunk 2 above)
Comment (ahasztag) [reply doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_customize_dfu.rst]: Fixed

**Review by maciejpietras — APPROVED** [review]

**Review by FrancescoSer — APPROVED** [review]

**Pull request merged**